src/prefect/events/schemas/automations.py

```diff
@@ -416,7 +416,7 @@ class AutomationCore(PrefectBaseModel, extra="ignore"):  # type: ignore[call-arg
     enabled: bool = Field(
         default=True, description="Whether this automation will be evaluated"
     )
-    tags: list[str] = Field(
+    tags: List[str] = Field(
```

Comment:
```suggestion
    tags: list[str] = Field(
```

Reply:
Ahh, the rest of that model is using the `typing` module types, I'll just change them all